pages/faq/troubleshooting/faq.md

```diff
@@ -143,6 +144,11 @@ There are a few options for devices that do not have an official device type on
 
 Power supply units (PSUs) are a critical component to any widespread production deployment, as mentioned in our [Going to Production][go-to-production] guide. When buying a PSU for your hardware, be sure to inspect and understand the board's power requirements carefully. Current requirements (measured in Amps) are just as important as the voltage requirements (measured in Volts). A PSU that provides a higher voltage than required may damage the board, but a supplied voltage that is too low may cause the board to malfunction or fail to boot. If the PSU isn't capable of providing the maximum current that the board demands, then the board might eventually fail to perform under CPU-intensive loads. As the load increases, more current is drawn than the PSU can supply, which could lead to brown-outs (i.e., the board unexpectedly resetting, causing filesystem corruption). When buying PSUs or using the official power adapters, proper research is recommended to prevent scenarios like these in a production deployment.
 
+##### Does {{ $names.company.lower }} have access to my device, source code and images?
+
+Device access is granted to a subset of {{ $names.company.lower }} employees to enable support and device troubleshooting. This access is controlled by ssh key access and only after access is granted to balena by the device fleet manager.
+Application source code and images are stored on {{ $names.company.lower }} backend servers with access limited only to administrative/operational staff and are not exposed to anyone outside of {{ $names.company.lower }}. It is also possible to bypass the {{ $names.company.lower }} builder entirely and push only pre-built artefacts, meaning that {{ $names.company.lower }} never has access to the code at any point.
```

Comment:
```suggestion
Device access is granted to a subset of {{ $names.company.lower }} employees to [enable support and device troubleshooting][support-access]. This access is controlled by ssh key access and only after access is explicitly granted to balena. 
Application source code and images are stored on {{ $names.company.lower }} backend servers with access limited only to administrative/operational staff and are not exposed to anyone outside of {{ $names.company.lower }}. It is also possible to bypass the {{ $names.company.lower }} builder entirely and push only pre-built artifacts, meaning that {{ $names.company.lower }} never has access to the code at any point.
```

Suggested a few changes @NatasaKyvetou 
You can add a link below to the support access document - https://www.balena.io/docs/learn/manage/support-access using the reference style links, you can read more about them here https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md#adding-links-to-the-docs